data-table/__tests__/table.js

```diff
@@ -485,6 +508,19 @@ describe('Interaction tests', () => {
         const input = label.getElementsByTagName("input")[0];
         expect(input.type).toBe("checkbox");
     });
+
+    test("Test the type is String", () => {
+        table.createDataTable({
+            'wrapperDivId': 'div-id',
+            'types': [Number, String]
```

Comment:
As above, consider
```suggestion
            'types': [String]
```

Reply:
TypeError: Cannot set property 'id' of null

      380 |         }
      381 | 
    > 382 |         field.id = constructInputFieldId(config.wrapperDivId, rowIndex, colIndex, fieldNum);
          |         ^

Reply:
somehow we have to put two types here for the testing